code/modules/projectiles/guns/_shared_ammo_objects.dm

```diff
@@ -35,6 +35,7 @@
 		COMSIG_ATOM_ENTERED = PROC_REF(on_cross),
 	)
 	AddElement(/datum/element/connect_loc, connections)
+	AddComponent(/datum/component/submerge_modifier, 10)
```

Comment:
Cute change but no mention in the pr desc or changelog.
also why is this in shared ammo objects, pain

Reply:
updated